Source/Core/Common/SettingsHandler.cpp

```diff
@@ -41,7 +41,7 @@ void SettingsHandler::SetBytes(Buffer&& buffer)
 
 std::string SettingsHandler::GetValue(std::string_view key) const
 {
-  constexpr char delim[] = "\r\n";
+  constexpr char delim[] = "\n";
```

Comment:
```suggestion
  constexpr char delim = '\n';
```

Reply:
That change breaks the next code line: `error: no matching function for call to ‘std::__cxx11::basic_string<char>::basic_string(const char&)’`

Reply:
Oh, I was pretty sure there was an overload for `char`... my bad.